src/screens/General/Dashboard.js

```diff
@@ -501,52 +508,36 @@ class Dashboard extends Component {
 		);
 	}
 
-	renderCommittees(item) {
+	calculateRankings() {
```

Comment:
Something like this is done in leaderboard. This function does not need to be in a component but could be put into utils in some file called leaderboard and imported wherever used. Would obviously need to pass in things as parameters and return same information. You can import firebase into the util

Reply:
Did you move this? I don't see it on the pr